lib/cli.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * CLI Argument Parsing / Handling
+ */
+
+const { program }  = require("commander");
+const package = require("../package.json");
+const fs = require("fs");
+const path = require("path");
+
+// Default Testdriver yaml file
+const DEFAULT_FILE = "testdriver/testdriver.yml";
+
+/**
+ * Parse the CLI Arguments.
+ * @param {string[]} args - List of string arguments (usually process.argv).
+ * @returns {Object} - An object containing the parsed command and file.
+ */
+const parseArgs = (args) => {
+  let userCommand = "";
+  let userFile = "";
+
+  program
+    .name("testdriverai")
+    .description("Generate and edit testdriver yaml files")
+    .version(package.version);
+
+  program
+    .command("init")
+    .description("Initialize A testdriver YAML file")
+    .argument("[file]", "File to use", DEFAULT_FILE)
+    .action((file) => {
+      userCommand = "init";
+      userFile = file
+    });
```

Comment:
The description here is not true and I don't think argument is supported?

`init` will send the user through prompts to set up their `env` and download the `testdriverai/quickstart-web` repo locally

Reply:
oh I should have checked what init actually did haha